packages/fluentui/react-northstar/src/themes/teams-high-contrast/colors.ts

```diff
@@ -204,8 +204,8 @@ export const colorScheme: ColorSchemeMapping = {
     foreground2: colors.white,
     foreground3: colors.black,
     background: colors.white,
-    background1: colors.white,
-    border: undefined,
+    background1: colors.black,
```

Comment:
I'm asking myself if there isn't any unintentional changes to other components with this change

Reply:
Same question for all of the existing colour scheme tokens that you've changed

Reply:
You're correct, instead of modifying these, there are new slots to add. :)

Reply:
Sorry I didn't quite get the conclusion, `background1` should or shouldn't be modified ?

Reply:
It should not be modified. Updating today.